mavros/src/lib/ftf_frame_conversions.cpp

```diff
@@ -64,6 +64,48 @@ using Matrix6d = Eigen::Matrix<double, 6, 6>;
 using Matrix9d = Eigen::Matrix<double, 9, 9>;
 
 
+Eigen::Matrix3d ecef_enu_rotation(const Eigen::Vector3d &map_origin)
```

Comment:
You don't have to move this out: Just create the same method `transform_static_frame` but with a new parameter, keeping the previous:
`Eigen::Vector3d transform_static_frame(const Eigen::Vector3d &vec, const Eigen::Vector3d &map_origin, const StaticTF transform)`